frontend/public/components/resource-quota.jsx

```diff
@@ -30,11 +124,77 @@ const Row = ({obj: rq}) => <div className="row co-resource-list__item">
   </div>
 </div>;
 
+const CapacityIcon = ({capacity}) => {
+  let capacityIconClass = 'pficon-unknown';
+  if (capacity === 0) {
+    capacityIconClass = 'fa-circle-thin co-m-resource-quota-empty';
+  } else if (capacity > 0 && capacity < .5) {
+    capacityIconClass = 'pficon-resources-almost-empty';
+  } else if (capacity >= .5 && capacity < 1){
+    capacityIconClass = 'pficon-resources-almost-full';
+  } else if (capacity === 1) {
+    capacityIconClass = 'pficon-resources-full';
+  } else if (capacity > 1) {
+    capacityIconClass = 'pficon-warning-triangle-o';
+  }
+  const classPrefix = capacityIconClass.startsWith('pficon') ? 'pficon' : 'fa';
+  return <i className={`${classPrefix} ${capacityIconClass}`} aria-hidden="true" />;
+};
+
+const getCapacity = (used, max, resourceType) => {
```

Comment:
This seems misnamed. Capacity usually means the total amount available. Maybe `getDecimalUsage` here and `UsageIcon` above?

Reply:
fixed